frontend/src/components/SelectHintColorizer.vue

```diff
@@ -30,7 +30,9 @@ export default {
   },
   watch: {
     hintColor (hintColor) {
-      this.applyHintColor(hintColor)
+      if (this && this.$el) {
+        this.applyHintColor(hintColor)
+      }
```

Comment:
in applyHintColor: use html class list

Reply:
done, added test for colorizer component